src/graphql/resolvers/businessInfoResolver.ts

```diff
@@ -4,16 +4,18 @@ import {
 	getExtractionProvider,
 } from '../../providers/extraction_providers/extractionProviderFactory';
 import { getInterpretationProvider, InterpretationConfig } from '../../providers/interpretation_providers';
-import { BusinessInferences, BusinessInfoInput } from '../generatedTypes';
+import { BusinessInferences, QueryGetBusinessInfoArgs } from '../generatedTypes';
 
-export async function getBusinessInfo(businessInfo: BusinessInfoInput): Promise<BusinessInferences> {
+export async function getBusinessInfo(_: any, args: QueryGetBusinessInfoArgs): Promise<BusinessInferences> {
 	if (!process.env.EXTRACTION_CONFIG) throw new Error('Undefined environment variable EXTRACTION_CONFIG');
 	if (!process.env.INTERPRETATION_CONFIG)
 		throw new Error('Undefined environment variable INTERPRETATION_CONFIG');
 
 	const extractionConfig: ExtractionConfig = JSON.parse(process.env.EXTRACTION_CONFIG);
 	const interpretationConfig: InterpretationConfig = JSON.parse(process.env.INTERPRETATION_CONFIG);
```

Comment:
A lot to unpack here:
1. Checking for correct configs seems like something that is commonly done... you can farm it out to something like a `withConfig(key: string): Promise<string>` that can do the error handling for you... then you can string them in a promise chain like: `return withConfig(process.env.A.then(configString => JSON.parse(configString)).then(AConfig => {doSomethingWithAConfig; return withConfig(process.env.B)}).then(...)...`
2. Alternative to 1, in the future you can check requirements/prerequisites using GraphQL Directives.
3. The thrown error will likely be shown to the caller of the graphql endpoint... who probably won't care about missing configurations... that message is targetted at the admin/maintainer, not the end user.

Reply:
This is related to #43.

Reply:
Deferred to #43